src/Directory.Build.props

```diff
@@ -24,7 +24,7 @@
     <!--Framework versions-->
     <UWPTargetPlatformVersion>10.0.14393.0</UWPTargetPlatformVersion>
     <UWPTargetPlatformMinVersion>10.0.14393.0</UWPTargetPlatformMinVersion>
-    <AndroidTargetFramework>monoandroid60</AndroidTargetFramework>
+    <AndroidTargetFramework>monoandroid81</AndroidTargetFramework>
```

Comment:
This is needed to avoid a warning about the ConstraintLayout library using a later target. This should not affect compatibility by itself - it is only the version of the SDK targeted at compile time and is independent of the supported minimum version.